geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java

```diff
@@ -203,8 +202,8 @@
 
     config = system.getConfig();
     try {
-      commandProcessor =
-          new OnlineCommandProcessor(system.getProperties(), cache.getSecurityService(), cache);
+      commandProcessor = cache.getService(CommandProcessor.class);
+      commandProcessor.init(cache);
```

Comment:
Is this a redundant call. The initialization is done before the service is returned

Reply:
Thanks for pointing this out, it should be fixed now.